### PR TITLE
Improve display of missing data on records

### PR DESCRIPTION
**Pull request by edwardhorsford**
HESA records are sometimes missing data that we'd normally require. Since this can't be modified by users, we'll style this differently than our standard blue missing style.

New style:
Screenshot 2022-04-29 at 16 47 53

PR updates the seeds so that these items are commonly missing from HESA records:
* Nationality
* ITT End date

I also tweaked the placement summary card so that it will *always* show at least two rows. If you're missing both placements it only shows one call to action link though.

A record missing one placement:
Screenshot 2022-04-29 at 16 52 20

A record missing both placements:
Screenshot 2022-04-29 at 16 52 38

A HESA record missing one placement:
Screenshot 2022-04-29 at 16 54 13

A HESA record missing both placements:
Screenshot 2022-04-29 at 16 53 51

### CODE REVIEW

File: app/views/records.html [hunk 1]
```diff
@@ -62,6 +62,8 @@
       classes: "app-status-box--filter-outdent",
       html: adminOnlyFilters
     }) }}
+    {% else %}
+      {% include "_includes/filter-panel/source.njk" %}
```
Comment (ralph-hawkins) [inline app/views/records.html]: I'm not sure this should be at the top filter for non-admin users?

File: app/views/_includes/filter-panel/source.njk [hunk 2]
```diff
@@ -22,11 +22,11 @@
...
       value: "DTTP",
       checked: checked(query.filterSource, "DTTP")
-    } if data.isAdmin and navActive !== "drafts",
+    } if navActive !== "drafts",
     {
       text: "Imported from HESA",
       value: "HESA",
       checked: checked(query.filterSource, "HESA")
-    } if data.isAdmin and navActive !== "drafts"
+    } if navActive !== "drafts"
```
Comment (ralph-hawkins) [inline app/views/_includes/filter-panel/source.njk:30]: Can we hide this from SCITTs?

**Review by ralph-hawkins — APPROVED** [review]

File: app/views/_includes/filter-panel/source.njk (hunk 2 above)
Comment (edwardhorsford) [reply app/views/_includes/filter-panel/source.njk:30]: If we only show if they have different sources, any reason they shouldn't get this filter?

(nb: a bit wary of too many differences between SCITTs and HEIs - I don't think we currently special case either of them?)

File: app/views/records.html (hunk 1 above)
Comment (edwardhorsford) [reply app/views/records.html]: Yes probably. Will move lower.

File: app/views/_includes/filter-panel/source.njk (hunk 2 above)
Comment (ralph-hawkins) [reply app/views/_includes/filter-panel/source.njk:30]: We show the same filters to everyone in the prototype. So it feels weird to always show this to SCITTS, given they'll never have HESA records. 

We could make the filters only show for relevant sources, but we'd end up with the same thing.

Comment (edwardhorsford) [reply app/views/_includes/filter-panel/source.njk:30]: Well we'd have apply vs manual for scitts.

**Pull request merged**